### DISCUSSION TITLE
Sins 2 E4X

### DISCUSSION THREAD

**Discussion by hardlecure** [GoaFan77/Enhanced4XModRepo#27] [Ideas]
Hi all. I love the e4x mod and have been using it for probably as long as it's been available. It's the only one I use honestly. I'm curious if you have plans to make it for Sins 2 as well. I'm also curious if you'd be able to add a feature that improves research speed if you build more than 8 military labs/research stations (with diminishing returns on each extra one built... like maybe an extra 8 provides 50% extra total research speed but an extra 8 on top of that is only 75% ).

**Comment by GoaFan77** [comment]
We are looking at modding Sins 2, but probably not with an E4X mod right a way. Sounds like there are 4 DLCs at least planned for Sins 2, and I suspect the vanilla game will continue to evolve rapidly in the post launch era. At this point I think its best we focus on trying to get E4X style improvements incorporated officially into the base game.
